src/libs/ValidationUtils.js

```diff
@@ -301,6 +302,18 @@ function isValidRoutingNumber(number) {
     return false;
 }
 
+/**
+ * Returns object with character limit errors. If any are valid,
+ * those errors get returned as empty strings.
+ *
+ * @param {Number} limit
+ * @param {Object[]} toBeValidated
+ * @returns {Object[]}
+ */
+function getCharacterLimitErrors(limit, toBeValidated) {
+    return _.map(toBeValidated, x => (x.length <= limit ? '' : Localize.translateLocal('personalDetails.error.characterLimit', {limit})));
```

Comment:
Let's just return `true` or `false` instead of the error itself as the error will always be the same for all.